packages/wallets/src/components/WalletListCardDropdown/WalletListCardDropdown.tsx

```diff
@@ -1,24 +1,34 @@
-import React, { ComponentProps, useCallback, useEffect, useMemo, useState } from 'react';
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import classNames from 'classnames';
 import { Trans, useTranslation } from 'react-i18next';
 import { useActiveWalletAccount, useWalletAccountsList } from '@deriv/api-v2';
 import { displayMoney } from '@deriv/api-v2/src/utils';
+import { LabelPairedChevronDownLgFillIcon } from '@deriv/quill-icons';
 import useWalletAccountSwitcher from '../../hooks/useWalletAccountSwitcher';
 import { THooks, TSubscribedBalance } from '../../types';
-import { WalletDropdown, WalletText } from '../Base';
+import reactNodeToString from '../../utils/react-node-to-string';
+import { WalletText, WalletTextField } from '../Base';
 import { WalletCurrencyIcon } from '../WalletCurrencyIcon';
 import './WalletListCardDropdown.scss';
 
-type WalletList = ComponentProps<typeof WalletDropdown>['list'] | undefined;
+type WalletList = {
+    currency?: string;
```

Comment:
can you get the type from the api hooks or something

Reply:
THooks.WalletAccountsList['wallet_currency_type']

Reply:
Updated